docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -76,6 +78,35 @@ The optional `{batch}` named query references the Batch of data under test. When
 
 :::
 
+## Source-to-target Expectations
+
+A source-to-target Expectation executes one SQL query for each of two Data Sources and compares their results for equality. This can be helpful for validating consistency between systems during data migration or regular data loading processes. Source-to-target Expectations can detect data drift introduced during the ETL process through discrepancies in schemas, counts, time windows, data types, and precision levels between Data Sources. Here are some examples of comparisons you can test:
+- Every row in table A matches every row in table B.
+- An aggregate metric of table A matches the same aggregate metric of table B.
+- An aggregate metric of table A matches a different aggregate metric of table B. (For example, the count of rows where X is true in table A matches the count of rows where Y and Z are true in table B.)
+
+To compare results for equality, each row returned by the query for the target Data Source will be compared to each row returned by the query for the source Data Source. When you configure a source-to-target Expectation, you set an **Expected percentage of identical rows**. The Expectation will fail if the percentage of identical rows between your two queries falls below this threshold.
```

Comment:
`Expected percentage of identical rows` <- `mostly`? I'd make this clear.

Reply:
I'd like to leave out this detail for now and see if we get questions about this before updating the docs to be explicit about how UI fields map to API parameters. This page doesn't cover any other UI field -> API parameter mappings so it feels weird to me to detail just this one unless we get questions about it / unless we refactor the page to cover both UI and API workflows for everything.